Makefile

```diff
@@ -145,11 +151,16 @@ interactive: up
 interactive:  ## start an interactive session using the project image (caution: R/W mounts source directory to /app)
 	docker run --rm -it -p 3000:3000 --name=$(CONTAINER_NAME_PREFIX)dev -e TANGO_HOST=$(TANGO_HOST) --network=$(NETWORK_MODE) \
 	       -v $(CURDIR):/app --volumes-from=$(CONTAINER_NAME_PREFIX)rsyslog-tmcprototype:rw nexus.engageska-portugal.pt/ska-docker/tango-java:latest /bin/bash
-#          -v $(CURDIR):/app --volumes-from=$(CONTAINER_NAME_PREFIX)rsyslog-tmcprototype:rw $(IMAGE_TO_TEST) /bin/bash
 
 down:  ## stop develop/test environment and any interactive session
 	docker ps | grep $(CONTAINER_NAME_PREFIX)dev && docker stop $(PROJECT)-dev || true
-	$(DOCKER_COMPOSE_ARGS) docker-compose down
+	$(DOCKER_COMPOSE_ARGS) docker-compose \
```

Comment:
Same comment as specified in up tag. Re-order starting and stopping of devices to resolve dependencies. (If required).

Reply:
Done.

Reply:
Commit ID: [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]